mailbag/__init__.py

```diff
@@ -16,6 +16,25 @@
 mailbagit_args = bagit_parser.add_argument_group("Mailbag")
 # add mailbag-specific args here
 mailbagit_args.add_argument("--foo", help="The foo argument, you know, that one")
+mailbagit_args.add_argument("--i","--input_type", help="input type MBOX/PST/IMAP/EML/PDF/MSG",nargs=1)
+mailbagit_args.add_argument("--input_path", help="input path MBOX/PST/IMAP/EML/PDF",nargs=1)
+mailbagit_args.add_argument("--o","--output", help="output type MBOX/PST/EML/PDF/WARC",nargs=1)
+mailbagit_args.add_argument("--exclude_input_file", help="excludes the original PST, MBOX, or EML file(s) from the mailbag",nargs='+')
+mailbagit_args.add_argument("--imap_host", help="the host for creating a mailbag from an IMAP connection",nargs=1)
+mailbagit_args.add_argument("--imap_password", help="password for creating a mailbag from an IMAP connection",nargs=1)
+mailbagit_args.add_argument("--exclude_folders", help="email folders to be excluded from the mailbag",nargs='+')
+mailbagit_args.add_argument("--exclude_messages", help="a line-separated text file of Message-IDs to be excluded from a mailbag",nargs='+')
+mailbagit_args.add_argument("--exclude_input", help="will also exclude email folders and/or messages from the input PST or MBOX file before including it in the mailbag",nargs='+')
```

Comment:
Some of these don't take input and are just Boolean switches. To do this we'll use `action='store_true'` instead of `nargs`.